util/src/test/java/io/grpc/util/OutlierDetectionLoadBalancerTest.java

```diff
@@ -1195,7 +1285,7 @@ private void generateLoad(Map<Subchannel, Status> statusMap,
       ClientStreamTracer clientStreamTracer = pickResult.getStreamTracerFactory()
           .newClientStreamTracer(null, null);
 
-      Subchannel subchannel = ((OutlierDetectionSubchannel) pickResult.getSubchannel()).delegate();
+      Subchannel subchannel = (Subchannel) pickResult.getSubchannel().getInternalSubchannel();
```

Comment:
This is confusing since `getInternalSubchannel()` specifically says it does NOT return a `Subchannel`.  Since you are using mocks and telling it to return what you want, it works, but would be better to get the value cleanly.

Reply:
It's sort of hacky, @ejona suggested this way somewhere in the comments, main purpose is to hide HealthCheckingSubchannel.